history/cultures/human_group.txt

```diff
@@ -81,6 +79,8 @@
 	#
 	discover_innovation = innovation_primogeniture
 	discover_innovation = innovation_cranes
+	#
+	discover_innovation = innovation_gryphon
```

Comment:
Quote from Grox: "It wasn't until WoW that we get to see Bronzebeard or Ironforge Gryphons."
Humans and Bronzebeards shouldn't have Gryphons at the beginning of the first war! 603 or 605 imho.